src/lib.rs

```diff
@@ -306,6 +307,18 @@ impl UUID {
             uuid: Uuid::from_u128(int),
         })
     }
+
+    #[getter]
+    fn is_safe<'py>(&self, py: Python<'py>) -> Bound<'py, PyAny> {
+        unsafe {
+            match SAFE_UUID {
```

Comment:
Can't we implement the logic in Rust? Instead of using unsafe with GIL  which will slow us down?

Reply:
I think we can return `*mut ffi::PyObject` directly, but I guess pyo3 will still bind it somewhere